packages/containerd/containerd-cri-base-json

```diff
@@ -0,0 +1,151 @@
+{
+    "ociVersion": "1.0.2-dev",
+    "process": {
+        "user": {
+            "uid": 0,
+            "gid": 0
+        },
+        "cwd": "/",
+{{#if settings.oci-defaults.capabilities}}
+        "capabilities": {
+            {{oci_defaults "capabilities" settings.oci-defaults}}
```

Comment:
TODO: build a test image and debug-print the 2nd param: is the setting path provided in the param blob?